pkg/kubectl/cmd/util/helpers.go

```diff
@@ -46,6 +46,7 @@ import (
 
 const (
 	ApplyAnnotationsFlag = "save-config"
+	AddOnLabel           = "kubernetes.io/cluster-service"
```

Comment:
why would people need to switch? I would expect the write-protect annotation to be added to the cluster service yaml files